README.md

```diff
@@ -73,3 +73,33 @@ Then run the following command in your bash prompt to download content to your l
 ```bash
 bundle exec jekyll contentful
 ```
+
+### Images
+
+With the help of imgix, we are using two strategies to increase performance while rendering images:
+
+- Pixellated placeholders
+- `srcset` and `sizes` attributes
+
+For more info on how this works, see the extended documentation in the [media repo](https://github.com/crdschurch/crds-media/blob/development/README.md#images).
+
+#### Examples
+
+**Inline Images**
+```html
+<img src="{{ page.image | imgix: site.imgix }}?{{ site.imgix_params.placeholder }}" sizes="{{ site.image_sizes.full_width }}" data-optimize-img>
```

Comment:
Rather than appending the query strings link this, could we clean up this implementation and just let the imgix filter do all the work?